include/runtime.iol

```diff
@@ -108,6 +108,12 @@ type Stats:void {
 		.systemLoadAverage:double //< System load average
 		.version:string //< OS version
 	}
+
+	.jvm:void{
```

Comment:
`jvm` is not very intuitive for me. What about `memory`? That way we don't need the `Mem` suffix in the subnodes.

```
memory {
  free:long
  used:long
  total:long
}
```

Reply:
the idea was to be able to get more JVM related info in a future but I can change it not a problem